generators/server/templates/src/main/java/package/security/SecurityUtils.java.ejs

```diff
@@ -135,6 +135,11 @@ public final class SecurityUtils {
      * If the current user has a specific authority (security role).
      * <p>
      * The name of this method comes from the {@code isUserInRole()} method in the Servlet API.
+<%_ if (authenticationType === 'oauth2') { _%>
+     * /!\ Call to this method won't check Oauth2 SCOPEs but only groups and roles.
```

Comment:
I believe the method name itself clarifies this part and the comment can be omitted